src/routes/[...slugs=myanimelist]/[id=int]/+page.svelte

```diff
@@ -36,4 +38,46 @@
         anime_cover={anime.cover}
         {anime_episodes}
     />
+{:else}
+    <container class="relative flex h-[80vh] w-full flex-col items-center justify-center md:h-[90vh] md:gap-[1vw]">
+        <radial-gradient
+            class="h-68 pointer-events-none absolute inset-0 z-10 hidden h-[40vw] w-[40vw] translate-x-[27vw] md:flex"
+            style="
+                background-image: radial-gradient(circle at center, rgba(255, 255, 255, 0.05) 0%, transparent 75%);
+            "
+        />
+        <four-zero-four class="flex items-center gap-5 text-[5rem] font-bold md:gap-[2vw] md:text-[7vw]">
+            <span>4</span>
+            <ghost-animate>
+                <Ghost class="w-20 animate-bounce md:w-[7vw]" />
+            </ghost-animate>
+            <span>4</span>
+        </four-zero-four>
+
+        <context class="flex max-w-[90%] flex-col items-center gap-5 md:max-w-[60vw] md:gap-[1.5vw] md:leading-[1.5vw]">
+            <status-code class="text-xl font-semibold text-primary-400 md:text-[1.5vw]">404 - Page not found!</status-code>
+            <status class="text-center text-sm font-semibold text-surface-50 md:text-[1.1vw] md:leading-[1.5vw]">
+                Uh-oh, looks like our
+                {#each "kokoro-chan".split("") as letter}
+                    <span class="inline-grid odd:text-warning-400 even:text-white">
+                        {letter}
+                    </span>
+                {/each}
+                is worked really hard for the past few days and now has fallen asleep. You can wait for her to wake up by looking at the status page, or come say hi to other fellow
+                {#each "kokoro-chan".split("") as letter}
+                    <span class="inline-grid odd:text-warning-400 even:text-white">
+                        {letter}
+                    </span>
+                {/each}
```

Comment:
Is there any way to avoid this duplicate `each` blocks:
```svelte
{#each "kokoro-chan".split("") as letter}
       <span class="inline-grid odd:text-warning-400 even:text-white">
                {letter}
       </span>
{/each}
```

Reply:
Nope